src/Filter/InAndNotInUtils.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace ipl\Sql\Filter;
+
+use ipl\Sql\Select;
+
+trait InAndNotInUtils
+{
+    /** @var string[]|string */
+    protected $column;
+
+    /** @var Select */
+    protected $value;
+
+    /**
+     * Get the columns of this condition
+     *
+     * @return string[]|string
+     */
+    public function getColumn()
+    {
+        return $this->column;
+    }
+
+    /**
+     * Set the columns of this condition
+     *
+     * @param string[]|string $column
+     *
+     * @return $this
+     */
+    public function setColumn($column): self
+    {
+        $this->column = $column;
+
+        return $this;
+    }
+
+    /**
+     * Get the value of this condition
+     *
+     * @return Select
+     */
+    public function getValue(): Select
+    {
+        return $this->value;
+    }
+
+    /**
+     * Set the value of this condition
+     *
+     * @param Select $value
+     *
+     * @return $this
+     */
+    public function setValue($value): self
```

Comment:
can the type declaration be used here?

Reply:
Argument type or return type declaration?

Reply:
argument type of course

Reply:
No you can't!

```php
( ! ) Fatal error: Declaration of ipl\Sql\Filter\InAndNotInUtils::setValue(ipl\Sql\Select $value): ipl\Sql\Filter\In must be compatible with ipl\Stdlib\Filter\Condition::setValue($value) in /usr/local/src/ipl-sql/src/Filter/InAndNotInUtils.php on line 56
```